### ISSUE TITLE
[Accessibility][Carousel] Previous buttons need an accessible label

### ISSUE THREAD

**Issue by Gajalakshmi0408** [adobe/aem-core-wcm-components#2069]
**Component : core/wcm/components/carousel/v1/carousel**
**Current behavior**
Previous/Next buttons are announced by screen-readers, but cannot be authored/modified on page since it is fetched from 
i18n value.

**Expected behavior/code**
To make the Previous/Next buttons to be authorable as part of the cq:dialog .

**Affected Users**
Screen reader users

**Possible Solution**
1. Make the Previous/Next buttons to be authorable. 
2. Introduce an aria-label attribute for Previous/Next buttons for Accessibility purpose.

**Comment by Gajalakshmi0408** [comment]
@gabrielwalt 
Can we make the Previous/Next button to be authorable so that authors will have flexibility to change it. 
With the current code structure, the screen reader will narrate as **Previous** & **Next** buttons. This might
not be understood by many screen reader users. If we have the authoring field/aria-labels for buttons, we can give the inputs like "Previous slide" and "Next Slide", which will be understandable to the screen-reader users.

**Comment by Gajalakshmi0408** [comment]
@gabrielwalt Do we have any update on this ticket? Please let me know for any further clarification/s needed from my end.

**Comment by Gajalakshmi0408** [comment]
@gabrielwalt Do we have any clarification needed/justification on this ticket please?

**Comment by gabrielwalt** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
To make the accessibility labels of the carousel component more customizable by the author, we'd need the following:
* Text inputs to set the "Previous", "Next", "Play", and "Pause" button labels.
* Text input to set the "Choose a slide to display" aria label.
* Checkbox to use the carousel names in the aria labels (instead of the "Slide 1...N" labels). This checkbox could be enabled by default for new carousels that are added to the page - but kept as is for existing content.

**Comment by vladbailescu** [comment]
Fixed by #2210
